app/controllers/users/piv_cac_authentication_setup_controller.rb

```diff
@@ -28,6 +27,10 @@ def delete
 
     private
 
+    def create_nonce
```

Comment:
should this duplicated *nonce*nse be in a service object?

Reply:
I'll put that into a concern since it accesses the `user_session` which probably needs the controller object context.